src/lib/ipsec/README.md

```diff
@@ -38,6 +38,15 @@ be a table with the following keys:
 * `spi` - “Security Parameter Index” as specified in RFC 4303.
 * `window_size` - *Optional*. Width of the window in which out of order packets
   are accepted. The default is 128. (`esp_v6_decrypt` only.)
+* `resync_threshold` - *Optional*. Number of consecutive packets allowed to
+  fail decapsulation before attempting re-synchronization. The default is
+  10,000. (`esp_v6_decrypt` only.)
+* `resync_attempts` - *Optional*. Number of attempts to re-synchronize
+  a packet that triggered re-synchronization. The default is 10.
+  (`esp_v6_decrypt` only.)
+* `auditing` - *Optional.* A boolean value indicating whether to enable or
+  disable “Auditing” as specified in RFC 4303. The default is `nil` (no
```

Comment:
Using nil as a boolean? Ew. (Not a blocker for this merge, but possibly worth poking upstream about.)